http/util.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
+/** Returns true if the etags match. Weak etag comparisons are handled. */
+export function compareEtag(a: string, b: string): boolean {
```

Comment:
Maybe name this file headers.ts?

Reply:
I prefer to keep it `util.ts` at this point as we don't have plan to provide exhaustive list of utilities about headers. Maybe let's reconsider it again when this module developed more.